docs/how-we-work.md

```diff
@@ -114,20 +118,28 @@ When finalizing your review:
-
-When you update Atom's `package.json`, make sure you wait for Atom to build before you merge your changes. In particular, we've had issues with snapshot tests. You can either do `apm publish pre` on the branch with the fix, then modify `package.json` in your local atom and try a `script/build`.  Or you can open a pull requests and let the CI tests run for you.
-
-When you've merged substantial new functionality, consider running `apm publish minor` and updating `package.json` on Atom's master branch outside of the Atom release cycle, to give the rest of the Atom team time to dogfood the change internally and weigh in with opinions.
-
-After shipping a minor version release for either of the above situations, create and push a release branch from that version's tag:
-
-```sh
-$ apm publish minor
-version 0.11.0
-$ git branch 0.11-releases && git push -u origin 0.11-releases
-```
-
-When you merge a fix for a bug, cherry-pick the merge commit onto to the most recent release branch, then run `apm publish patch` and update `package.json` on the most recent beta release branch on the `atom/atom` repository. This will ensure bug fixes are delivered to users on Atom's stable channel as part of the next release.
-
-When you merge a fix for a **security problem**, a **data loss bug**, or fix a **crash** or a **lock-up** that affect a large portion of the user population, cherry-pick the merge commit onto the most recent beta _and_ stable release branches of atom/github that contain the bug, then run `apm publish patch` on both and update `package.json` on the affected release branches on the `atom/atom` repository. Consider advocating for a hotfix release of Atom to deliver these fixes to the user population as soon as possible.
+At the end of each development sprint:
+
+1. _In your atom/github repository:_ run `apm publish preminor` to create the first prerelease version or `apm publish prerelease` to increment an existing prerelease version. Note the generated version number and ensure that it's correct. If the currently deployed version is `v0.19.2`, the first prerelease should be `v0.20.0-0`; if the existing prerelease is `v0.20.0-0`, the next prerelease should be `v0.20.0-1`.
+2. _In your atom/atom repository:_ create a new branch and edit `package.json` in its root directory. Change the version of the `"github"` entry beneath `packageDependencies` to match the prerelease you just published.
+3. _In your atom/atom repository:_ Run `script/build --install`. This will update Atom's `package-lock.json` files and produce a local development build of Atom with your prerelease version of atom/github bundled.
```

Comment:
should this be `script/build --ci --install`?

Reply:
Not in this case! `--ci` can speed up installation substantially, but it only works it `package-lock.json` is up to date, which it won't be here because we just edited `package.json`.

If anyone _else_ checks out this atom/atom pull request with the lockfile changes included, then they can add `--ci` to get a boost to their build time.

Reply:
ohhhh, that makes sense!  Thanks for clarifying.